package.json

```diff
@@ -44,7 +44,7 @@
   },
   "devDependencies": {
     "eslint": "^8.34.0",
-    "prettier": "^2.7.1",
-    "typescript": "^4.7.4"
+    "prettier": "^2.8.4",
+    "typescript": "^4.9.5"
```

Comment:
I think we have issues with decorators with typescript above a specific version. On versions 4.8.x or above. That's one of the main reasons most repos only go to 4.7.4.

https://devblogs.microsoft.com/typescript/announcing-typescript-4-8/#decorators-are-placed-on-modifiers-on-typescripts-syntax-trees

Reply:
hmm didn't see any obvious issues when I tested it in adjudication service but I didn't try deploying to integration to confirm, I'll give that a quick go

Reply:
no type errors from the decorators, was able to deploy to integration and tested the S2S call, everything behaved as expected

Reply:
It would be great to test in a hex service where the decorators aren't coming from neo-framework 🙏